CodyJones4235/NewWorldMarket_Frontend
# Posts, Auth, Providers

Description of PR that completes issue here...

## Changes

-Added all functionality for Registering, Logging in, creating a post, and viewing posts

## Testing

1. Make sure your server is running
2. checkout all the links in the navbar and make sure they switch to the corresponding page
3. Check `http://localhost:3000/Register` and create a user for yourself
3. Check `http://localhost:3000/Login` and login to the user you just created
3. Check `http://localhost:3000/CreatePost` and create a new post and it should display after hitting sent
4. Check `http://localhost:3000/Posts` to see all post and the post you created above
